src/test/ui/issues/issue-79690.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0080]: it is undefined behavior to use this value
+  --> $DIR/issue-79690.rs:26:1
+   |
+LL | const G: Fat = unsafe { Transmute { t: FOO }.u };
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ type validation failed: encountered read of part of a pointer at .1.<deref>.size.foo
```

Comment:
This error message doesn't seem super clear to me. In fact, I'm not sure if *I* understand it! Will it be understandable to people getting this error, or can it be improved somehow?